label_studio/ml_models/serializers.py

```diff
@@ -0,0 +1,22 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
+
+from ml_models.models import ModelInterface
+from rest_framework import serializers
+from users.serializers import UserSimpleSerializer
+
+
+class CreatedByFromContext:
```

Comment:
Is this just required so we can de-normalize the user info of the `created_by` user? What does the `requires_context` attribute do?

Reply:
Correct yeah instead of displaying `user: 1` we will use the `1` ID to get all field of a user, makes it easier for frontend to work with

Reply:
Good point about requires context, I copied that from other examples in our codebase but will check